### PR TITLE
Add RowMap to go along with ColumnMap

### PR DESCRIPTION
**Pull request by zombor**

### CODE REVIEW

File: gocumber.go [hunk 1]
```diff
@@ -38,6 +38,19 @@ func ColumnMap(table Table) map[string]string {
...
 
+func RowMap(table Table) map[string]string {
+	result := make(map[string]string)
+
+	header := table.Rows()[0]
+	for _, row := range table.Rows()[1:] {
+		for i, key := range header {
+			result[key] = row[i]
+		}
+	}
```
Comment (actualizeself) [inline gocumber.go]: Is this going to work for when you have more than one row in the table? it seems like result[key] is going to keep getting overwritten until the last row?

Comment (zombor) [inline gocumber.go]: Yeah. `RowMap` should return a `[]map[string]string` instead.

Comment (cbandy) [inline gocumber.go]: I would something named `Map` to return a single `map`.

Comment (zombor) [inline gocumber.go]: Changed in 21426ed.

Comment (zombor) [inline gocumber.go]: Open to suggestions on a new name.

File: gocumber.go [hunk 2]
```diff
@@ -38,6 +38,23 @@ func ColumnMap(table Table) map[string]string {
 	return result
 }
 
+func RowMap(table Table) []map[string]string {
```
Comment (cbandy) [inline gocumber.go]: I think this be plural; `RowMaps` or maybe `MapRows`...

Comment (zombor) [inline gocumber.go]: Changed in de0aa80.

**Comment by actualizeself** [comment]
:mountain_bicyclist:

**Pull request merged**